<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingDialog</class>
 <widget class="QDialog" name="BuildingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>82</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>82</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>82</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Building</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Building the network</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
